Getting_Started.md

```diff
@@ -408,6 +430,15 @@ To enable these workflows, you (or the repository owner) need to configure the f
             Ensure this service account has the "Firebase Hosting Admin" role in your Google Cloud project's IAM settings.
     *   **When to set:** If you want the GitHub Actions to automatically deploy your application to Firebase Hosting. If this secret is not set (even if `FIREBASE_CONFIG_JSON` is), the build may complete, but the deployment step will be skipped.
 
+3.  **Additional Secrets for Functions Deployment** (Optional)
+    *   These secrets are used by the `deploy-functions.yml` workflow and may be required by your Firebase Functions:
+    *   **`OPENAI_API_KEY`**: OpenAI API key for AI/ML functionality
+    *   **`DEEPINFRA_API_KEY`**: DeepInfra API key for additional AI services
+    *   **`GITHUB_TOKEN`**: GitHub personal access token for repository operations
```

Comment:
Also note, if you want to use other providers, you would need to add their keys here as well- and the whole thing would cross reference to the keys listed in https://github.com/The-AI-Alliance/agent-lab-ui/blob/main/src/constants/agentConstants.js